addon/utils/performance-now.js

```diff
@@ -1,5 +1,5 @@
 export default function performanceNow() {
-  if ('performance' in window) {
+  if ('performance' in window && typeof window.performance.now === 'function') {
```

Comment:
would be great to make things fastboot compatible here, by adding `typeof Fastboot === 'undefined'` as a condition. Otherwise this will cause a hard failure (fastboot server crash)